include/nighthawk/client/factories.h

```diff
@@ -45,7 +45,7 @@ class BenchmarkClientFactory {
                                     Envoy::Stats::Scope& scope,
                                     Envoy::Upstream::ClusterManagerPtr& cluster_manager,
                                     Envoy::Tracing::HttpTracerSharedPtr& http_tracer,
-                                    absl::string_view cluster_name,
```

Comment:
Optional suggestion: consider an extra argument "sink_stat_prefix" or some such? I can see this is convenient and makes sense in the context of this change, but from a conceptual PoV it makes less sense to tie BenchmarkClient to a worker number (instead of the cluster/upstream, scope, etc it uses)?

Reply:
Done.